### DISCUSSION TITLE
Do I really need 8444 port open?

### DISCUSSION THREAD

**Discussion by damdinsharav** [Chia-Network/chia-blockchain#3077] [Q&A]
I live in apartment where ISP provides internet to many apartment under single router and I can't configure port forwarding.

Currently Chia app says it's fully synced, also I see occasionally 1 or  2 plots were eligible for farming message in debug log. Although I haven't won any xch.

Is it must to port forward(NAT) 8444 to my chia node?

**Comment by Cast39** [comment]
No you don't have to do this. It is just recommended, because this can speed up the syncing process but since you are already synced you don't have to enable port forwarding

**Comment by GeorgePapageorgakis** [comment]
I guess it must not be mandatory because it raises security issues. You could easily get attacked by an experienced hacker.
